assets/js/modules/analytics-4/utils/is-zero-report.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Report utilities.
+ *
+ * Site Kit by Google, Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Checks whether the report data is empty or not.
+ *
+ * @since n.e.x.t
+ *
+ * @param {Object} report Report data object.
+ * @return {(boolean|undefined)} Returns undefined if in the loading state, true if the report has no data or missing data, otherwise false.
```

Comment:
This function doesn't have the "loading" state. We can probably rephrase it to be something like this: `Returns TRUE if the report data has only zero values, FALSE if there is at least one non-zero value, or undefined if the report data is not resolved yet.`

Reply:
As above, I've applied this change to both this and the UA version.